web/demo/index.html

```diff
@@ -18,65 +18,90 @@
 -->
 <!doctype html>
 <html>
+<head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Apache Annotator (incubating) demo</title>
-  <link rel="stylesheet" href="https://unpkg.com/awsm.css/dist/awsm.min.css">
+  <link rel="stylesheet" href="../style.css"/>
   <style>
-    blockquote {
-      font-style: italic;
-    }
     a[href^="#selector"]:before {
-      content: '📌';
-      font-size: 75%;
+      content: '📌 ';
     }
-    mark {
-      background-color: rgba(255, 255, 0, 0.5);
-      outline: 0.1px solid rgba(255, 100, 0, 0.8);
+
+    #selectable, #corpus {
+      padding: 1em;
+      border: 1px solid lightgrey;
     }
+
     #parsed {
-      height: 15rem;
+      height: 15em;
       overflow: scroll;
+      background: #f8f8f8;
+      color: #666666;
+      resize: vertical;
     }
   </style>
+  <script src="bundle.js"></script>
```

Comment:
Safe to toss an `async` attribute on this if you're gonna move it up to head.

Reply:
Good point; I made it `defer` instead, because with `async` we would have to check for the race condition (it could miss the `DOMContentLoaded` event).